crawl-ref/docs/options_guide.txt

```diff
@@ -2021,6 +2023,18 @@ tile_force_overlay = false
         the same number of lines. Increase the view_max_height option if you
         find yourself with unused screen estate.
 
+tile_overlay_col = #646464
```

Comment:
```suggestion
tile_overlay_colour = #646464
```

Please give this option a verbose name, since it is adjacent to other options that involve the word column.